cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageImpl.kt

```diff
@@ -79,6 +85,18 @@ private class DataStorageImpl(
                 .apply()
         }
 
+    /**
+     * Parameter that accesses applies value from ccpaConsentStatus directly
+     *
+     * By default - false (if there are no ccpaConsentStatus in data storage or applies value is null)
+     */
+    override val ccpaApplies: Boolean
```

Comment:
@bohdan-go-wombat what about the `set`? If I understood it right, we access `ccpaApplies` by reaching out to the ccpa consent object, but can still set its value independently. Is that right?

Reply:
when I refactored I made `ccpaApplies` `val`, since we no longer need to update the `mirroring` value, and we will just update the local `ccpaConsentStatus` object (the one that is stored in the data store and being used in our app)

Reply:
getter is needed to have the easier access to the `ccpaApplies`, for example rather then calling something like:
`dataStorage.ccpaConsentStatus?.let { verifiedCcpaConsentStatusString ->
            check { JsonConverter.converter.decodeFromString<CcpaCS>(verifiedCcpaConsentStatusString) }
                .getOrNull()
                ?.applies`
we will call:
`dataStorage.ccpaApplies`